Kernel/API/Syscall.h

```diff
@@ -132,7 +132,9 @@ namespace Kernel {
     S(dump_backtrace)         \
     S(dbgputch)               \
     S(dbgputstr)              \
-    S(watch_file)             \
+    S(create_iwatcher)        \
+    S(iwatcher_add_watch)     \
+    S(iwatcher_remove_watch)  \
```

Comment:
Please use `inode_watcher` instead of `iwatcher`.

Reply:
We had talked about this above with Ali, and decided that `inode_watcher_add_watch` would be a bit on the verbose side, but I'll change them.